### PR TITLE
fix status check with proc-ns-ps

### PR DESCRIPTION
**Pull request by zhangguanzhang**
should used the busybox ps command
Fixes: https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic/issues/141#issuecomment-1094272670
Signed-off-by: zhangguanzhang <zhangguanzhang@qq.com>

### CODE REVIEW

File: luasrc/controller/unblockneteasemusic.lua [hunk 1]
```diff
@@ -24,7 +24,7 @@ end
 
 function act_status()
 	local e = {}
-	e.running = luci.sys.call("ps -w |grep unblockneteasemusic |grep app.js |grep -v grep >/dev/null") == 0
+	e.running = luci.sys.call("busybox ps -w |grep unblockneteasemusi[c] |grep /app.js >/dev/null") == 0
```
Comment (1715173329) [inline luasrc/controller/unblockneteasemusic.lua:27]: Adding busybox here is enough.

Comment (zhangguanzhang) [reply luasrc/controller/unblockneteasemusic.lua:27]: > Adding busybox here is enough.

used the regex could cuased less command

**Comment by zhangguanzhang** [comment]
all had done

**Comment by 1715173329** [comment]
Merged into f3fb36789917023ffb2d3f35798d16877e2279fc.

**Pull request closed without merging**